### ISSUE TITLE
posting to Facebook is excluded by default on iOS

### ISSUE THREAD

**Issue by DamianAntonowicz** [jguertl/SharePlugin#51]
I can see in code (https://github.com/jguertl/SharePlugin/blob/master/Share/Share.Plugin.iOSUnified/ShareImplementation.cs) that posting to Facebook is excluded by default on iOS version of plugin. I would like to know why?

**Comment by jamesmontemagno** [comment]
It was removed because sharing to Facebook was removed by Facebook so it doesn't do anything as they no longer implement the API from testing and the interwebs as a whole.

**Comment by DamianAntonowicz** [comment]
Sorry, but I don't understand your answer. Sharing on Facebook is excluded by default on iOS version of plugin. However there is no such case on Android version of plugin. I've used following code:
`_Share.Share(new ShareMessage
            {
                Text = "text",
                Title = "title",
                Url = "https://github.com"
            },
            new ShareOptions
            {
                ExcludedUIActivityTypes = new ShareUIActivityType[0]
            });`

This code allows to share on Facebook on iOS. I've tested this and it works - I've posted message to my wall.

I still don't understand why sharing on Facebook is excluded by default on iOS. Could you be more specific why there is such code on iOS implementation?

**Comment by jamesmontemagno** [comment]
Perhaps Facebook fixed it. I will add back in. Previously no text at all would even show up, which is why we had a request to default it off.

Android there is no API to control share targets as they are generic.
